pkg/cli/admin/node/logs.go

```diff
@@ -258,37 +287,44 @@ func (o LogsOptions) RunLogs() error {
 		req := client.Get().RequestURI(path).
 			SetHeader("Accept", "text/plain, */*").
 			SetHeader("Accept-Encoding", "gzip")
-		if o.Path == "journal" {
-			if len(o.UntilTime) > 0 {
-				req.Param("until", o.UntilTime)
+		if o.Path == "journal" || len(o.Query) > 0 {
+			if len(o.Until) > 0 {
```

Comment:
It is a bit more difficult to convert `Since` to `SinceTime` and `Until` to `UntilTime` as `Since` and `Until` could be a ISO date or a relative time. i.e. `YYYY-MM-DD HH:MM:SS` or `(+|-)[0-9]+(s|m|h|d)`. So the relative time to ISO 3339 format is especially tricky and I wanted to avoid that. So my suggestion we keep them separate and mark `Since` and `Until` for deprecation in the future.

Reply:
Yes, type incompatibility between `Since` and `SinceTime` sounds good to me, but what will happen if any CI script relies on `Since`;

```sh
$ oc adm node-logs --since
```
will it continue working?

Reply:
Yes that will continue. Please see line 294.

Reply:
Fixed.